components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/index.jsp

```diff
@@ -107,10 +108,10 @@
                 %>
                 <tr>
                     <td width="50%"><a
-                            href="claim-view.jsp?store=<%=UserCoreConstants.INTERNAL_DOMAIN%>&dialect=<%=claimDialectDTO[i].getDialectURI()%>"><%=claimDialectDTO[i].getDialectURI()%>
+                            href="claim-view.jsp?store=<%=Encode.forUriComponent(UserCoreConstants.INTERNAL_USERSTORE)%>&dialect=<%=Encode.forUriComponent(claimDialectDTO[i].getDialectURI())%>"><%=Encode.forHtmlContent(claimDialectDTO[i].getDialectURI())%>

```

Comment:
Previously value of param store used to be 'UserCoreConstants.INTERNAL_DOMAIN' which is hardcoded. You have changed that to UserCoreConstants.INTERNAL_USERSTORE. Can you please comment on why it was changed. Anyhow since these hardcoded values does not contain special characters and since they are trusted, it's not necessary to encode them.